pkg/gadgets/dns/tracer/tracer.go

```diff
@@ -167,7 +167,101 @@ var pktTypeNames = []string{
 	"KERNEL",
 }
 
-func parseDNSEvent(rawSample []byte) (ret string, pktType string) {
+// List taken from:
+// https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4
+var qTypeNames = []string{
```

Comment:
```suggestion
var qTypeNames = map[int]string{
```

I'm not sure what the difference is but I think the second one is more readable.

Reply:
More readable and I also think it takes less spaces, thank you!